lib/timelineService.js

```diff
@@ -99,9 +99,10 @@ function checkUrl(url) {
 
 function extractUUID(link) {
```

Comment:
Is there a util file where that `extractUUID` function can be defined? It's repeated in several files.